### DISCUSSION TITLE
Vast.AI integration

### DISCUSSION THREAD

**Discussion by MRiabov** [skypilot-org/skypilot#4314] [Ideas]
Hello SkyPilot,
I've just found your framework, but I'm using a vast.ai service. It's a service that allows you to rent GPUs so cheap no other cloud provider can beat them.
It's not a promo, but just take a look at their pricing: https://cloud.vast.ai/create/, especially at interruptible (spot). You can get an H100 rented for 0.5-1.2$/hour, if you find the right deal (just 0.5$!!!). This is, of course, spot. Gcloud hosts the same for 4.2 - almost 4-8 time more expensive.

It's so cheap because it's non-professional renting, for example some university may rent their GPU installation over a weekend and still be profitable. There are hundreds of RTX4090 rigs too. 4090 rigs go around 0.1-0.15$/hour.

Many users would want such a cheap pricing; and I'll probably start migrating with skypilot because I want to a) simplify my spot instance work for future inference and b) use Google TPUs because they have great prices too.

Could you please consider implementing a vast.ai interface into the system? 
Cheers.

(I'm not selling, it's just a truly great service)

**Comment by concretevitamin** [comment]
Being worked on in https://github.com/skypilot-org/skypilot/issues/2930!

**Comment by Michaelvll** [comment]
This is now merged in #4365! Great thanks to @kristopolous. Closing this discussion now.
